instrumentation/opentelemetry/init.go

```diff
@@ -208,12 +208,13 @@ func createCaCertPoolFromFile(certFile string) *x509.CertPool {
 // Init initializes opentelemetry tracing and returns a shutdown function to flush data immediately
 // on a termination signal.
 func Init(cfg *config.AgentConfig) func() {
-	return InitWithSpanProcessorWrapper(cfg, nil)
+	return InitWithSpanProcessorWrapper(cfg, nil, versionInfoResourceAttributes)
 }
 
 // InitWithSpanProcessorWrapper initializes opentelemetry tracing with a wrapper over span processor
 // and returns a shutdown function to flush data immediately on a termination signal.
-func InitWithSpanProcessorWrapper(cfg *config.AgentConfig, wrapper SpanProcessorWrapper) func() {
+func InitWithSpanProcessorWrapper(cfg *config.AgentConfig, wrapper SpanProcessorWrapper,
```

Comment:
Just curious, what is the advantage of passing around the func rather than the slice? 

Secondary is, can the versionInfo be const?

Reply:
I could make it a var which will be an attribute array.

Reply:
Done